config/config.keys.json

```diff
@@ -14,5 +14,6 @@
   "nyt_states": "covidapi:nyt_states",
   "nyt_USA": "covidapi:nyt_usa",
   "apple_all": "covidapi:apple_all",
-  "gov_countries": "covidapi:gov_countries"
+  "gov_countries": "covidapi:gov_countries",
+  "ebola": "covidapi:ebola"
```

Comment:
I don't see any reason why ebola has `covidapi:ebola` as resource key
we can either have `covidapi:` for covid disease related stuff and `ebola:` for ebola related stuff